perfzero/lib/perfzero/utils.py

```diff
@@ -242,17 +242,12 @@ def run_command(cmd, shell=True):
   Returns:
     Tuple of the command return value and the standard out in as a string.
   """
-  logging.debug('Executing command: %s', cmd)
+  logging.debug('Executing command: {}'.format(cmd))
   p = subprocess.Popen(cmd, stdout=subprocess.PIPE,
                        stderr=subprocess.STDOUT, shell=shell)
-  stdout = ''
-  exit_code = None
-  while exit_code is None:
-    exit_code = p.poll()
-    line_str = p.stdout.readline().decode('utf-8')
-    logging.debug(line_str)
-    stdout += line_str
-  return exit_code, stdout
+  stdout, _ = p.communicate()
```

Comment:
Sorry I didn't realize this benefit of the existing implementation until now.

When we do `docker build...` in setup.py, the command will take a long time (> 10 minutes) to execute. It is probably better experience to keep printing the output from `docker build...` during this period. Otherwise, it is hard for user tell whether something goes wrong if there is no output for more than 10 minutes. Does this sound reasonable?